src/new-client/src/app/lib/types/index.tsx

```diff
@@ -116,13 +126,19 @@ type Session = {
   services: string[];
 };
 
+// TODO: Decide where this info should live and move it there
+const clinicalServices = [{ label: 'RSV (Adult)', value: 'RSV:Adult' }];
```

Comment:
Agreed :) This PR is only moving this type from being declared in the wizard to being declared here (because the wizard is a client component but I'm now consuming it in a server component)